### ISSUE TITLE
QueryEngine doesn't perform link traversal out of the box

### ISSUE THREAD

**Issue by mrkvon** [comunica/comunica-feature-link-traversal#56]
#### Issue type:

- :bug: Bug

____
#### Description:

When importing `QueryEngine` from [@comunica/query-sparql-link-traversal v0.0.2-alpha.2.0](https://www.npmjs.com/package/@comunica/query-sparql-link-traversal) in a [minimal Node (v17.9.0) + Typescript project](https://github.com/mrkvon/comunica-evaluation-node/blob/main/src/index.ts), the link traversal doesn't work. I.e. only the sources that are provided beforehand get visited.

```ts
import { QueryEngine } from '@comunica/query-sparql-link-traversal'

;(async () => {
  const myEngine = new QueryEngine()

  const bindingsStream = await myEngine.queryBindings(
    `
      PREFIX foaf: <http://xmlns.com/foaf/0.1/>
      PREFIX mypod: <https://mrkvon.solidcommunity.net/profile/card#>
      PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
      SELECT DISTINCT ?interest ?label
      WHERE {
        mypod:me foaf:topic_interest ?interest.
        ?interest rdfs:label ?label.
        FILTER(lang(?label)='en')
      }`,
    {
      sources: ['https://mrkvon.solidcommunity.net/profile/card'],
      lenient: true,
    },
  )
  // Do something with the results, but there are no results
})()
```

On the other hand, when we set up the engine as follows, the link traversal works as expected:

```ts
import { QueryEngineFactory } from '@comunica/query-sparql-link-traversal'

;(async () => {
  const myEngine = await new QueryEngineFactory().create({
    configPath:
      'https://raw.githubusercontent.com/comunica/comunica-feature-link-traversal/master/engines/query-sparql-link-traversal/config/config-default.json',
  })

  // The same query as above, or other .......
  // Do something with the results, and yay, we have some!
})()
```

____
#### Environment:

@comunica/query-sparql-link-traversal v0.0.2-alpha.2.0 (installed with `yarn add @comunica/query-sparql-link-traversal`
Node v17.9.0
Arch Linux

What else do you need to know?

**Comment by mrkvon** [comment]
I just tried to clone [this repository](https://github.com/comunica/comunica-feature-link-traversal/tree/c2a1ecff978a22ceb2c934a2a0c0a5291b812954), `yarn`, and run
```sh
node engines/query-sparql-link-traversal/bin/query.js https://mrkvon.solidcommunity.net/profile/card "
PREFIX foaf: <http://xmlns.com/foaf/0.1/>
PREFIX mypod: <https://mrkvon.solidcommunity.net/profile/card#>
PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
SELECT DISTINCT ?interest ?label
WHERE {                                  
  mypod:me foaf:topic_interest ?interest.
  ?interest rdfs:label ?label.
  FILTER(lang(?label)='en')
}" --lenient
```
And it _does_ perform the link traversal as expected

**Comment by mrkvon** [comment]
@Maximvdw That's awesome. This `npm run prepare` fixes my issue, too. 🚀

When a new package is released, let me know, I can test it and then this may be good to close. 🎉

**Comment by mrkvon** [comment]
> Running `npm run prepare` to recompile the config fixes the issue - so I think a new build would fix it
> 
> EDIT: To test, I tried it in my `node_modules\@comunica\query-sparql-link-traversal` directory

With this fix, I can use the library in React, too. 🎉

**Comment by rubensworks** [comment]
@mrkvon new alpha build released: `0.0.2-alpha.3.0`

Thanks for testing @Maximvdw!
